website/source/docs/providers/azure/index.html.markdown

```diff
@@ -33,7 +33,7 @@ resource "azure_instance" "web" {
 
 The following arguments are supported:
 
-* `settings_file` - (Optional) The path to a publish settings file used to
+* `settings_file` - (Required) Contents of a valid `publishsettings` file, used to
```

Comment:
agh, that's actually a mistake I suppose, since you can do the file or the _other two things_. I was modeling off the Google one, thanks for pointing that out! Fixed in a7543de